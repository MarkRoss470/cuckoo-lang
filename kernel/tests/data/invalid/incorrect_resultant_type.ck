data Invalid : Prop where
  | c : Prop