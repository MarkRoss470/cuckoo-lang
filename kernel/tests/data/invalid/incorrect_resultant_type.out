ERROR at tests/data/invalid/incorrect_resultant_type.ck 2:9-2:13: Invalid resultant type for constructor. Constructor c should result in an application of Invalid, but it results in Sort 0